shopfloor/services/zone_picking.py

```diff
@@ -577,7 +605,9 @@ def _scan_source_package(
             if message:
                 return (
                     self._list_move_lines(
-                        self.zone_location, sublocation=package.location_id or False
+                        self.zone_location,
+                        sublocation=sublocation,
+                        package=package,
```

Comment:
The sublocation optional argument should be dropped. It is useless
```suggestion
                        sublocation,
                        package=package,
```

Reply:
IMO, it is still needed because _list_move_lines will call _response_for_select_line which needs to know if the sublocation has to be passed in the response to the front-end.

Reply:
What I mean is that I find it cleaner to only have one location argument in _list_move_lines and give the right location in the call. Instead of
```
def fn(location, sublocation):
  location = sublocation or location
  ...

fn(zone_location, sublocation)
```
i would prefer ```
def fn(location):
   ...

fn(sublocation or zone_location)
```
Here we want to list the move lines on the package.location_id. The zone_location serves nothing in this call. So it would be:
```self._list_move_lines(
                        package.location_id,
                        package=package)
```

But that's a detail, it changes nothing to the behavior and result